content/ch-algorithms/teleportation.ipynb

```diff
@@ -2454,14 +2455,41 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3. Simulating the Teleportation Protocol <a id='simulating'></a>"
+    "## 3. Jacklyn sends Vivian Roses: An Analogy <a id='analogy'></a>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 3.1 How Will We Test the Protocol on a Quantum Computer? <a id='testing'></a>"
+    "Now that we know the quantum teleportation protocol, I want to take a step back and make the protocol more relatable with an analogy.\n",
```

Comment:
Let's try and stick with the multi-author language. Perhaps "let's take a step back and..."

Reply:
I will incorporate that, thank you.